src/libs/NumberUtils.js

```diff
@@ -0,0 +1,22 @@
+import CONST from "../CONST";
+
+function rand64() {
```

Comment:
Let's add a jsdoc here that explains that this generates a positive 64 bit number and returns it as a `string` and that this is used for generating transaction IDs. Otherwise looks good!

Reply:
s/transaction IDs/IDs

(we are about to reuse this for reportActionIDs)

Reply:
Agree with JSDocs and yes, the usage will be more than just transactionIDs